tests/go/less_atts.sgrep

```diff
@@ -0,0 +1,3 @@
+func $F(...) {
```

Comment:
what does atts mean? And why did you prefix this test with a less? It looks more like a dots_xxx.
The spirit of the less_xxx is to say that a pattern can have less details that the concrete file.
For example writing the pattern class A { ...} will also match code where the code has the extra detail
about inheritnace like class A extends B {}. We do not force the user to write class A extends $X { ...} .

Reply:
Move to tests/go/dots_func_attributes.sgrep to clarify!